features/mbedtls/targets/TARGET_NUVOTON/TARGET_NUC472/sha/sha_alt_hw.h

```diff
@@ -16,12 +16,6 @@
 #ifndef MBEDTLS_SHA_ALT_HW_H
 #define MBEDTLS_SHA_ALT_HW_H
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
-
 #if defined(MBEDTLS_SHA1_C) || defined(MBEDTLS_SHA256_C) || defined(MBEDTLS_SHA512_C)
 #if defined(MBEDTLS_SHA1_ALT) || defined(MBEDTLS_SHA256_ALT) || defined(MBEDTLS_SHA512_ALT)
```

Comment:
I woudl suggest splitting this file into 1 file per module as with the software ones...

Reply:
@andresag01 SHA1/SHA256/SHA512 share the same SHA H/W, so these code are placed together.